index.html

```diff
@@ -43,19 +43,19 @@
         <div class="station-info">
             <div class="entry bikes">
                 <span>vélos<br/>disponibles</span>
-                <img class="entry--logo" alt="" src="img/velib.svg" />
+                <img class="logo" alt="" src="img/velib.svg" />
```

Comment:
So it is the same issue here: you are inside an `.entry` so you can name your subblocks with `.entry--logo`.